Formula/binutils.rb

```diff
@@ -19,14 +19,17 @@ class Binutils < Formula
   keg_only :shadowed_by_macos, "Apple's CLT provides the same tools"
 
   uses_from_macos "bison" => :build
-  uses_from_macos "texinfo" => :build
   uses_from_macos "zlib"
 
   link_overwrite "bin/gold"
   link_overwrite "bin/ld.gold"
   link_overwrite "bin/dwp"
 
   def install
+    # Workaround https://sourceware.org/bugzilla/show_bug.cgi?id=28909
+    touch "gas/doc/.dirstamp", mtime: Time.utc(2022, 1, 1)
+    make_args = OS.mac? ? [] : ["MAKEINFO=true"] # for gprofng
```

Comment:
How useful even is `gprofng`? I'm a little keen on disabling it so we can also skip `bison`.

Reply:
No idea. Though at least `bison` is lighter.

`texinfo` dependency is indicitive of a bug, since doc files should be pregenerated.

Reply:
Does `bison` appear elsewhere in the Linux bootstrap dep tree? If not, we should just get rid of it.

Reply:
It does not.